db/app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const { handlePsqErrors, handleCustomErrors, handleServerErrors } = require("./errors");
+const { getAllTopics } = require("./app.controller");
+
+const app =express()
+
+app.use(express.json())
+
+app.get('/api/topics', getAllTopics)
+
+
+
+app.use(handlePsqErrors);
+app.use(handleCustomErrors);
+app.use(handleServerErrors);
```

Comment:
Following TDD, you should only add the code to pass the current tests. If these aren't needed for the current tests, then they should be added yet.